internal/pkg/cache/cache.go

```diff
@@ -0,0 +1,62 @@
+package cache
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/adrg/xdg"
+)
+
+var (
+	cacheFolderPath = xdg.CacheHome + "/stackit"
```

Comment:
you can use the std library: https://pkg.go.dev/os#UserCacheDir
then we can remove the `xdg` dependency